src/bitsea/Sat/aveSat.py

```diff
@@ -58,27 +58,33 @@ def argument():
 args = argument()
 
 from pathlib import Path
-from typing import List
+from typing import List, Iterable
 from bitsea.commons.Timelist import TimeList
 from bitsea.commons.time_interval import TimeInterval
 from datetime import datetime
 from bitsea.postproc import masks
 import numpy as np
 import bitsea.Sat.SatManager as Sat
+from sys import exit
 
 from bitsea.utilities.mpi_serial_interface import get_mpi_communicator
 
 
 def aveSat(*,
-inputdir: Path,
+inputfiles: Iterable[Path],
+datetimes: List[str],
 outputdir: Path,
 mesh: str,
 varnames = List[str],
```

Comment:
I think that this should be `varnames : List[str]`

Reply:
(y)